c++/misc.hpp

```diff
@@ -215,6 +217,30 @@ void writetable(const std::vector<std::pair<T1, T2>> &re, std::string filename,
   for (const auto & [x, y] : re) F << x << " " << y << std::endl;
 }
 
+template <typename T>
+Eigen::Matrix<T,Eigen::Dynamic,Eigen::Dynamic> ublas_to_eigen(ublas::matrix<T> m){
+  Eigen::Map<Eigen::Matrix<T,Eigen::Dynamic,Eigen::Dynamic, Eigen::RowMajor>> m_eigen(m.data().begin(),m.size1(),m.size2());
+  return m_eigen;
+}
+
+template <typename T>
+Eigen::Matrix<T,Eigen::Dynamic,1> ublas_to_eigen(ublas::vector<T> m){
+  Eigen::Map<Eigen::Matrix<T,Eigen::Dynamic,1>> m_eigen(m.data().begin(),m.size(),1);
+  return m_eigen;
+}
+
+// template<typename T, int N, int M>
```

Comment:
To se ne deluje?

Reply:
V testih sem delal te funkcije, ker se ti glavni headeri predolgo compileajo, potem pa sem te pozabil kopirat

Reply:
ok